README.md

```diff
@@ -1,25 +1,41 @@
 # Learningpath API
  [![Build Status](https://travis-ci.org/NDLANO/learningpath-api.svg?branch=master)](https://travis-ci.org/NDLANO/learningpath-api)
 
-API for Learningpaths from NDLA
+API for learningpaths from NDLA.
 
-# Building and distribution
+## Usage
+Creates, updates, deletes and returns a Learningpath. Implements elastic search for search within the learningpath database.
 
-## Compile
+#### Licenses
+Returns a list of licenses with the possibility of filtering on the license key.
+
+#### Api access
+To interact with the api, you need valid security credentials; see [Access Tokens usage](https://github.com/NDLANO/auth/blob/master/README.md).
+
+To write data to the api, you need write role access. This is only accessible in [learningpath-frontend](https://learningpath-frontend.staging.api.ndla.no) today.
+
+For a more detailed documentation of the API, please refer to the [API documentation](https://staging.api.ndla.no).
+
+## Developer documentation
+
+### Compile
     sbt compile
 
-## Run tests
+### Run tests
     sbt test
 
-## Run integration tests
+### Run integration tests
     sbt it:test
 
-### IntegrationTest Tag and sbt run problems
+#### IntegrationTest Tag and sbt run problems
 Tests that need a running elasticsearch outside of component, e.g. in your local docker are marked with selfdefined java
-annotation test tag  ```IntegrationTag``` in ```/ndla/article-api/src/test/java/no/ndla/tag/IntegrationTest.java```. 
+annotation test tag  ```IntegrationTag``` in ```/ndla/article-api/src/test/java/no/ndla/tag/IntegrationTest.java```.
+
 As of now we have no running elasticserach or tunnel to one on Travis and need to ignore these tests there or the build will fail.  
+
 Therefore we have the
- ```testOptions in Test += Tests.Argument("-l", "no.ndla.tag.IntegrationTest")``` in ```build.sbt```  
+ ```testOptions in Test += Tests.Argument("-l", "no.ndla.tag.IntegrationTest")``` in ```build.sbt```
+
 This, it seems, will unfortunalty override runs on your local commandline so that ```sbt "test-only -- -n no.ndla.tag.IntegrationTest"```
```

Comment:
Slett hele avsnittet f.o.m. This, it seems, will blablabla til og med "This should be solved better" ... tror ikke det stemmer lengre.